src/randoop/SerializableExpectedExceptionChecker.java

```diff
@@ -3,8 +3,14 @@
 import java.io.ObjectStreamException;
 import java.io.Serializable;
 
-import randoop.util.Reflection;
-
+import randoop.types.TypeNames;
+
+/**
+ * Serialized form of {@link ExpectedExceptionChecker} allowing tests to
+ * be serialized.
+ * 
+ * Also see <code>ExpectedExceptionChecker.writeReplace</code>.
```

Comment:
No such class.  I think you mean ExpectedExceptionCheck.  Please change to a `@link` and make sure that "make javadoc" runs without any warnings.